packages/block-library/src/code/index.js

```diff
@@ -39,8 +39,7 @@ export const settings = {
 	transforms: {
 		from: [
 			{
-				type: 'pattern',
-				trigger: 'enter',
+				type: 'enter',
```

Comment:
I think this is a typo?
```suggestion
				type: 'prefix',
```

Reply:
No, it should be enter, it's a different transform.